sdk/identity/azure-identity/azure/identity/_credentials/azd_cli.py

```diff
@@ -111,7 +115,8 @@ def get_token(
         :param str scopes: desired scope for the access token. This credential allows only one scope per request.
             For more information about scopes, see
             https://learn.microsoft.com/entra/identity-platform/scopes-oidc.
-        :keyword str claims: not used by this credential; any value provided will be ignored.
+        :keyword str claims: additional claims required in the token, such as those returned in a resource provider's
```

Comment:
No breaking change here, I assume ?

```suggestion
        :keyword str or None claims: additional claims required in the token, such as those returned in a resource provider's
```

Reply:
The signature defines `claims: Optional[str] = None`. Is it necessary to specify `str or None` again in the docstring? 

(I ask because we have other instances using `:keyword str claims`. If required, I will update all those instances accordingly.)

Reply:
Thats the pattern Ive always used to have it in there. Does it show up in the editor or apiview now?